dpgen/data/tools/create_random_disturb.py

```diff
@@ -214,6 +223,7 @@ def create_disturbs_abacus_dev(fin, nfile, dmax=1.0, etmax=0.1, ofmt="abacus", d
         stru_d['cells'] = cell_new
 
         convert_mat = np.linalg.inv(cell).dot(cell_new)
+        convert_mat = np.linalg.inv(cell)*cell_new
```

Comment:
This was a bug fixed #815 .

It seems that at some point you did not track the changes with git. I suspect there are other unwanted changes in this PR. 

I suggest you checking out a new branch based on the latest `devel` branch, applying **your changes** on it and then resubmitting the PR.

Reply:
Ok, thanks for your suggestion. I will track the latest **devel** branch and resubmit the PR.